### DISCUSSION TITLE
Something seems off...

### DISCUSSION THREAD

**Discussion by Duoquote** [community/community#134399] [npm]
When I search on npm with "react log" keyword and sorted by popularity, there are lots of spam repositories, I mean like 100+ repositories. Some of them even has 500k+ or even 1.5M downloads too:

![screencapture-npmjs-search-2024-08-02-12_35_40](https://github.com/user-attachments/assets/93f83321-251c-4ca6-9bbf-60f4db5eadc2)
![image](https://github.com/user-attachments/assets/1e5fa56d-da0d-4ef9-a14e-e81b42eb24d7)
![image](https://github.com/user-attachments/assets/27984074-f29c-4d02-b720-846cf544f299)

Some of them date back to 3 months too...

**Comment by grandmas-favorite** [comment]
Hi @Duoquote! 

Is this a new phenomenon for you or have you noticed this in the past as well?

**Reply by Duoquote** [reply]
It never occurred to me, it just happened today when I search for "react log" and sorted by popularity.

**Reply by leobalter** [reply]
Thanks for reporting this, @Duoquote! I'll bring it to our teams and plan for the best measures to remediate and prevent it from happening again.
